scenarios/reproducers/validated-architecture-shiftstack.yml

```diff
@@ -0,0 +1,159 @@
+#   post_apply_stage_INDEX
+#
+# cifmw_deploy_architecture_stopper:
+
+# What about some tempest?
+cifmw_run_tests: true
+cifmw_run_tempest: true
+cifmw_tempest_tests_allowed:
+  - tempest.scenario.test_network_basic_ops.TestNetworkBasicOps
+
+# This will instruct libvirt_manager to create 3 compute and
+# the ansible-controller, consuming the networks created and
+# managed by devscripts.
+# Note that the "osp_trunk" network is the equivalent of the
+# "private_net" in the CI layout, and will hold the VLAN for
+# network isolation.
+# The "ocpbm" network is managed by devscripts as well, and
+# provides access to Internet. This will be the equivalent of the
+# "public network" as seen in CI.
+cifmw_use_libvirt: true
+cifmw_libvirt_manager_configuration:
+  networks:
+    osp_trunk: |
+      <network>
+        <name>osp_trunk</name>
+        <forward mode='nat'/>
+        <bridge name='osp_trunk' stp='on' delay='0'/>
+        <ip family='ipv4'
+        address='{{ cifmw_networking_definition.networks.ctlplane.network |
+                    ansible.utils.nthhost(1) }}'
+        prefix='{{ cifmw_networking_definition.networks.ctlplane.network |
+                   ansible.utils.ipaddr('prefix') }}'>
+        </ip>
+      </network>
+  vms:
+    compute:
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
+        }}
+      amount: "{{ [cifmw_libvirt_manager_compute_amount|int, 3] | max }}"
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 200
+      memory: 50
+      cpus: 8
+      nets:
+        - ocpbm
+        - osp_trunk
+    controller:
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
+        }}
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - ocpbm
+        - osp_trunk
+    ocp:
+      amount: 3
+      admin_user: core
+      image_local_dir: "/home/dev-scripts/pool"
+      disk_file_name: "ocp_master"
+      disksize: "105"
+      xml_paths:
+        - /home/dev-scripts/ocp_master_0.xml
+        - /home/dev-scripts/ocp_master_1.xml
+        - /home/dev-scripts/ocp_master_2.xml
+      nets:
+        - osp_trunk
+
+
+## devscript support for OCP deploy
+cifmw_use_devscripts: true
+cifmw_devscripts_config_overrides:
+  worker_memory: 16384
+  worker_disk: 100
+  worker_vcpu: 10
+  num_extra_workers: 0
+# Required for egress traffic from pods to the osp_trunk network
+cifmw_devscripts_enable_ocp_nodes_host_routing: true
```

Comment:
ok, this is a the new param to enable the firewall - no need for a full copy-paste...

Reply:
Moved the var to shift-on-stack-overrides.yml as suggested